ai-decision-services/ai-ad-dpp-engine/run-tests/run-tests.md

```diff
@@ -0,0 +1,280 @@
+Lab 8: A simple data processing, training and inferencing example
+===
+
+## Training
+
+To make sure you have everything setup well in training, you can upload the training data to the **training-data-bucket**. If you see a Data Flow run is triggered is successful, and under Anomaly Detection a model is trained successfully, then the training path is working.
+
+Fundamentally, you should see processed data in csv under your **staging-bucket**:
+
+![](./images/Exp1.png)
+
+And under your **results-bucket**, your model information will be written to a file named **model_info.** It should look like this:
+
+```java
+{
+    "model_ids": [
+        {
+            "model_id": "ocid1.aianomalydetectionmodel.oc1.phx.amaaaaaaor7l3jiavbwqxnwiqvikmrdjyeaq7e2bvrpbh3y5apq4iv4xmfga",
+            "columns": [
+                "timestamp",
+                "sensor_01",
+                "sensor_02",
+                "sensor_03",
+                "sensor_04",
+                "sensor_05",
+                "sensor_06",
+                "sensor_07",
+                "sensor_08",
+                "sensor_09",
+                "sensor_10",
+                "sensor_11",
+                "sensor_12",
+                "sensor_13"
+            ]
+        }
+    ]
+}
+```
+
+Lastly, you should also see your model trained under **Anomaly Detection**. You can check the FAP and other details here:
+
+![](./images/Exp2.png)
+
+## Inferencing
+
+Similarly, to make sure you have everything setup correctly in inferencing, you can upload the inferencing data to the **inferencing-data-bucket**. If you see a Data Flow run is triggered is successful, and in resulting bucket you see the result of anomaly detection, then the inferencing path is working. The application assumes you have performed training prior to inference. If this is not the case, ensure that the model\_info file mentioned previously exists in **results-bucket** in the same format.
+
+After a successful run, you should see something like the following in your **results-bucket**:
```

Comment:
Consider rewording - After a successful run, the **results-bucket** should be populated as shown in the screenshot below....